sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/xml/XmlInferSchema.scala

```diff
@@ -39,6 +40,8 @@ class XmlInferSchema(options: XmlOptions, caseSensitive: Boolean)
     extends Serializable
     with Logging {
 
+  private val decimalParser = ExprUtils.getDecimalParser(options.locale)
```

Comment:
a qq - Does this parser function the same as our [XSD](https://src.dev.databricks.com/databricks/runtime/-/blob/sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/xml/XSDToSchema.scala?L89) conversion?

Reply:
If not, let's make a followup to match.

Reply:
This parser converts a decimal string to DecimalType. The one in XSDToSchema converts XSD type to DecimalType.